.ci/containers/contributor-checker/check-contributor.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+if [ -z "$GITHUB_TOKEN" ]; then
+    echo "Did not provide GITHUB_TOKEN environment variable."
+    exit 1
+fi
+if [ $# -lt 1 ]; then
+    echo "Usage: $0 pr-number"
+    exit 1
+fi
+PR_NUMBER=$1
+set -x
+
+USER=$(curl -H "Authorization: token ${GITHUB_TOKEN}" \
+  "https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/${PR_NUMBER}" | jq .user.login)
+
+ASSIGNEE=$(curl -H "Authorization: token ${GITHUB_TOKEN}" \
```

Comment:
Ok so I just remembered that of course PRs can have assignees but they can also have review requests, and those are two different things. What should we do here? I know I typically do review request instead of assign but I don't really care.

(https://developer.github.com/v3/pulls/review_requests/)

Reply:
I think it's more like "assign" that "request review" - since if you don't know what to do with the code you can request that someone else review it.  But if you want it changed I'm happy to!

Reply:
No strong preference here. Poll the team?